requirements.txt

```diff
@@ -1,9 +1,9 @@
-aiofiles>=0.8.0
-async_lru~=1.0
-galaxy_importer>=0.4.5
-GitPython>=3.1.24
-jsonschema~=4.4
-packaging
-pulpcore>=3.18
-PyYAML
-semantic_version
+aiofiles>=0.8.0,<0.9
+async_lru>=1.0,<1.1
+galaxy_importer>=0.4.5,<0.5
+GitPython>=3.1.24,<3.2
+jsonschema>=4.4,<4.7
+packaging>=21,<22
+pulpcore>=3.20,<3.25
+PyYAML<6.0
+semantic_version>=2.9,<2.11
```

Comment:
What motivated the lower bound here?

Reply:
`semantic_version` added python 3.7+ support in 2.9, so it should be the lowest we allow for a python 3.8 project.